core/arch/arm32/kernel/tee_ta_manager.c

```diff
@@ -625,6 +625,9 @@ static TEE_Result tee_ta_load(const kta_signed_header_t *signed_ta,
 #if defined(CFG_SE_API)
 	ctx->se_service = NULL;
 #endif
+#if !defined(CFG_TEE_CORE_NO_TA_TRACE)
+	ctx->trace = true;
+#endif
```

Comment:
Is the idea to enable/disable it runtime? I'm thinking about the check done in tee_svc_sys_log(..) further down.